salt/modules/virt.py

```diff
@@ -1937,6 +1940,21 @@ def init(
 
         By default, the value will ``"hd"``.
 
+    :param numatune:
+        The optional numatune element provides details of how to tune the performance of a NUMA host via controlling NUMA
+        policy for domain process. The optional ``memory`` element specifies how to allocate memory for the domain process
+        on a NUMA host. ``memnode`` elements can specify memory allocation policies per each guest NUMA node. The definition
+        used in the dictionary can be found at :ref:`init-cpu-def`.
```

Comment:
hmm, actually `nodeset` `mode` are coming from `init-cpu-def`. They are somehow connected.

Reply:
I am still convinced that the `numatune` parameter description shouldn't be shared with the cpu one, just for the sake of clarity. Otherwise you need to have clear references to the `init-cpu-def` sub section you mention. This starts to grow big, let's make it easy for the users.